### PR TITLE
refactor: use tune components on proposal page

### PR DESCRIPTION
**Pull request by samuveth**
### Summary

For consistency I've refactored modal and block components on the proposal page to use tune components, to match the changes from the Boost PR.

- [x] @ChaituVR I couldn't find any spaces still using Poap, do you know any?

| Before                | After                 |
|-----------------------|-----------------------|
|  Screenshot 2024-03-19 at 4 54 25 in the afternoon | Screenshot 2024-03-19 at 4 54 18 in the afternoon |
| Screenshot 2024-03-19 at 4 53 05 in the afternoon | Screenshot 2024-03-19 at 4 53 11 in the afternoon |
| Screenshot 2024-03-19 at 4 53 39 in the afternoon | Screenshot 2024-03-19 at 5 07 24 in the afternoon |
| Screenshot 2024-03-19 at 4 55 05 in the afternoon | Screenshot 2024-03-19 at 4 55 01 in the afternoon |
| Screenshot 2024-03-19 at 4 55 29 in the afternoon | Screenshot 2024-03-19 at 4 55 25 in the afternoon |

### CODE REVIEW

**Comment by ChaituVR** [comment]
@samuveth I found this for example https://snapshot.org/#/vote.airswap.eth/proposal/0x6fd2804ea9d303dae7693c1a81895246ece2022ce40a8e29ea0025ecd1f089bd

File: src/components/ModalPostVote.vue [hunk 1]
```diff
@@ -64,68 +63,69 @@ onMounted(() => {
 
           <template v-else>
             <h3 v-text="$t('proposal.postVoteModal.defaultTitle')" />
-            <p class="italic" v-text="$t('proposal.postVoteModal.tips.1')" />
+            <p class="italic" v-text="'Thank you for your participation!'" />
```
Comment (samuveth) [inline src/components/ModalPostVote.vue:66]: Not really related but added this to postvote modal:
image

Comment (ChaituVR) [reply src/components/ModalPostVote.vue:66]: Maybe better to update postVoteModal.tips.1 no?

Comment (ChaituVR) [reply src/components/ModalPostVote.vue:66]: Also italic look bit weird, better to keep it normal, but not too important

File: src/plugins/hal/components/CustomBlock.vue [hunk 2]
```diff
@@ -12,7 +12,7 @@ const halLogoUrl = computed(
 </script>
 
 <template>
-  <BaseBlock :title="$t('hal.title', { spaceName: space.name })">
```
[the remark this answers is not in the document]
Comment (ChaituVR) [reply src/plugins/hal/components/CustomBlock.vue:15]: Example proposal https://snapshot.org/#/bancornetwork.eth/proposal/0xbf21276c0a61590a1a76dd9aed326dca4277b4909493de1768ddcff924e2783f

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/components/SpaceProposalInformation.vue [hunk 3]
```diff
@@ -34,13 +34,7 @@ const symbols = computed((): string[] =>
...
             class="flex"
           >
-            <span
-              v-tippy="{
-                content: symbol
-              }"
-            >
-              <AvatarSpace :space="space" :symbol-index="symbolIndex" />
-            </span>
+            <AvatarSpace :space="space" :symbol-index="symbolIndex" />
```
Comment (ChaituVR) [inline src/components/SpaceProposalInformation.vue:37]: Now tooltip doesn't work for me, should be fine?

**Review by ChaituVR — APPROVED** [review]
tAck

File: src/components/SpaceProposalInformation.vue (hunk 3 above)
Comment (samuveth) [reply src/components/SpaceProposalInformation.vue:37]: Yeah I removed it, doesn't seem to be needed

**Pull request merged**